integration-tests/cypress/integration/api/recordings/recording_processing.ts

```diff
@@ -1224,7 +1323,7 @@ describe("Recordings - processing tests", () => {
 
     it("Recordings in other states not picked up for processing", () => {
       const recording14 = TestCreateRecordingData(templateRecording);
-      recording14.processingState = "FINIHED";
+      recording14.processingState = "FINISHED";
```

Comment:
Use the enum variants instead of strings to avoid typos etc.

Reply:
done and cast invalid values to accepted type where needed for test